solidity/contracts/bridge/Bridge.sol

```diff
@@ -1611,6 +1836,30 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         return info;
     }
 
+    function fraudSlashingAmount() external view returns (uint256) {
+        return frauds.slashingAmount;
+    }
+
+    function fraudNotifierRewardMultiplier() external view returns (uint256) {
+        return frauds.notifierRewardMultiplier;
+    }
+
+    function fraudChallengeDefeatTimeout() external view returns (uint256) {
+        return frauds.challengeDefeatTimeout;
+    }
+
+    function fraudChallengeDepositAmount() external view returns (uint256) {
+        return frauds.challengeDepositAmount;
+    }
```

Comment:
What if we put all these 4 functions into a single one named `fraudChallengeParameters`? Would it reduce the contract size? 

We could use the same trick in `RandomBeacon` in case we need more contract space to implement authorizations for T staking contract integration. cc @nkuba

Reply:
It will probably require creating the structure with these parameters so that only one thing is returned from the function (otherwise I would have to return 4 parameters).

Also this structure should probably replace parameters from the Frauds.Data structure so that we don't duplicate them:
```
        ///  The amount of stake slashed from each member of a wallet for a fraud.
        uint256 slashingAmount;
        /// The percentage of the notifier reward from the staking contract
        /// the notifier of a fraud receives. The value is in the range [0, 100].
        uint256 notifierRewardMultiplier;
        /// The amount of time the wallet has to defeat a fraud challenge.
        uint256 challengeDefeatTimeout;
        /// The amount of ETH in wei the party challenging the wallet for fraud
        /// needs to deposit.
        uint256 challengeDepositAmount;
```

@lukasz-zimnoch Are you ok with that? You wanted to avoid grouping them like that (pithing Parameters structure).

Reply:
We already have places where we return multiple params in similar cases (https://github.com/keep-network/tbtc-v2/blob/be73ffa655de9704ad660f461170147640f17911/solidity/contracts/bridge/Bridge.sol#L405) so grouping is not a hard requirement. If you think it will improve readability and code quality in this case, let's do it.

Reply:
My suggestion was about implementing a function similar to the one @lukasz-zimnoch referenced. From the contract size perspective, the biggest problem is that we have separate public functions.

We could have something like this instead:
```
function getFraudParameters() external view returns(
  uint256 slashingAmount,
  uint256 notifierRewardMultiplier,
  uint256 challengeDefeatTimeout,
  uint256 challengeDepositAmount
) {
  slashingAmount = frauds.slashingAmount;
  notifierRewardMultiplier = frauds.notifierRewardMultiplier;
  challengeDefeatTimeout = frauds.challengeDefeatTimeout;
  challengeDepositAmount = frauds.challengeDepositAmount;
}
```

Reply:
Done in 410c30553b051a465b0495507cbe58b81d90795e.
It decreased the size of `Bridge` a little bit: `23.181` -> `23.073`.